packages/@o3r/schematics/project.json

```diff
@@ -14,7 +14,25 @@
         "script": "postbuild"
       },
       "dependsOn": [
-        "compile"
+        "^build",
+        "compile",
+        "build-builders"
+      ]
+    },
+    "prepare-build-builders": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "prepare:build:builders"
+      },
+      "dependsOn": []
+    },
+    "build-builders": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "build:builders"
+      },
+      "dependsOn": [
+        "prepare-build-builders"
```

Comment:
why do you  override the nx.json version ? (it should be dependsOn compile and prepare-build-builders)

Reply:
Because here I do not necessary need that build-builders to be done after compile. Do you think we can have a problem if build builders and compile are executed parallelly ?